includes/classes/class-helpers.php

```diff
@@ -147,14 +137,7 @@ public static function get_option_as_array( $option_name ) {
 
 		if ( is_array( $option ) && ! empty( $option ) ) {
 			return $option;
-		} else {
-			return array();
-		}
-
-		if ( is_string( $option ) ) {
-			return array( $option );
```

Comment:
In this scenario, the `is_string` condition was never reachable - something that became apparent once the previous condition was simplified.